src/config/HeaderList.ts

```diff
@@ -0,0 +1,121 @@
+export const HeaderList = {
+    products: [
+        {
+            title: "Push Notifications",
+            subtitle: "Web3 native notification alerts",
+            srcref: "inbox-icon-on",
+            srcrefoff: "inbox-icon-off",
+            href: ""
```

Comment:
Add appropriate links

Reply:
Will do that - checking the correct links with design team